documentation/en/api-experimental-user-facing-v2-docs/api-v2-experimental.md

```diff
@@ -0,0 +1,1219 @@
+# Filecoin V2 APIs
```

Comment:
Is this going the be the new home of public facing docs on Filecoin v2 APIs or are we planning to use this as a scratch-space and bubble up the changes manually to notion?

Reply:
No.  This is just a staging area so can more easily review changes, at least for when "more likely to make a mistake" people like me are involved.  (Someone can also see revisions in Notion - it's just harder there to comment on the diff).

I made a note about this in https://github.com/filecoin-project/lotus/pull/13051/commits/bbd32817386a869a0c0df367e7fb75fb3ba1059b .  Resolving.